test/org/zaproxy/zap/extension/pscanrules/PasswordAutocompleteScannerUnitTest.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2016 The ZAP Development Team
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrules;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.parosproxy.paros.network.HttpMalformedHeaderException;
+import org.parosproxy.paros.network.HttpMessage;
+
+public class PasswordAutocompleteScannerUnitTest extends PassiveScannerTest {
+
+	@Override
+	protected PasswordAutocompleteScanner createScanner() {
+		return new PasswordAutocompleteScanner();
+	}
+
+	private HttpMessage createHttpMessage(String body) throws HttpMalformedHeaderException {
+        HttpMessage msg = new HttpMessage();
+        msg.setRequestHeader("GET https://www.example.com/test/ HTTP/1.1");
+        msg.setResponseHeader("HTTP/1.1 200 OK\r\n");
+        msg.setResponseBody(body);
+        return msg;
+	}
+
+	@Test
+	public void noForm() throws HttpMalformedHeaderException {
+        HttpMessage msg = createHttpMessage("<html><body></body></html>");
+        rule.scanHttpResponseReceive(msg, -1, this.createSource(msg));
```

Comment:
Not in this pull request IMHO.
Since that's an aesthetic change I guess it should be stated somewhere (dev guidelines?), so it's consistently applied.